packages/zoe/src/zoeService/zoeStorageManager.js

```diff
@@ -117,7 +122,7 @@ export const makeZoeStorageManager = (
     'zoeMintBaggageSet',
   );
   for (const issuerBaggage of zoeMintBaggageSet.values()) {
-    zoeMintBaggageSet(issuerBaggage);
+    prepareIssuerKit(issuerBaggage);
```

Comment:
Surprised to see `zoeMindBaggageSet` become `prepareIssuerKit`. Digging, I see that `zoeMintBaggageSet` just calls `provideDurableSetStore`. Neither sounds like it is doing a job similar to `prepareIssuerKit`.